Annotato/Annotato/Persistence/PersistenceService/LocalPersistenceService/LocalDocumentsPersistence.swift

```diff
@@ -57,14 +57,9 @@ struct LocalDocumentsPersistence {
     }
 
     func deleteDocument(document: Document) -> Document? {
-        guard let deletedDocument = localDocumentEntityDataAccess
-            .delete(documentId: document.id, document: document) else {
-            AnnotatoLogger.error("When deleting document.",
-                                 context: "LocalDocumentsPersistence::deleteDocument")
-            return nil
-        }
+        localDocumentEntityDataAccess.delete(documentId: document.id, document: document)
```

Comment:
The error is removed because delete now returns nothing right? I think we should still return something and log an error here if it's nil like all the other methods in this file.